history/units/SWE_1936_naval_mtg.txt

```diff
@@ -36,7 +36,7 @@
 			ship = { name = "HSwMS Sverige" definition = heavy_cruiser equipment = { ship_hull_cruiser_coastal_defense_ship = { amount = 1 owner = SWE version_name = "Sverige Class" } } }				
 			ship = { name = "HSwMS Drottning Victoria" definition = heavy_cruiser equipment = { ship_hull_cruiser_coastal_defense_ship = { amount = 1 owner = SWE version_name = "Sverige Class" } } }				
 			
-			ship = { name = "HSwMS Gotland" definition = light_cruiser equipment = { ship_hull_cruiser_2 = { amount = 1 owner = SWE version_name = "Gotland Class" } } }				
+			ship = { name = "HSwMS Gotland" definition = heavy_cruiser equipment = { ship_hull_cruiser_2 = { amount = 1 owner = SWE version_name = "Gotland Class" } } }				
```

Comment:
My God, thank you. I spent more time than I am willing to admit trying to figure out what was causing those bugs to show up in the debugger.

Reply:
Glad my OCD regarding them being listed as light cruisers in the intel ledger helped haha!